db/seeds.rb

```diff
@@ -1,7 +1,128 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+# This file should contain all the record creation needed to seed the database
+# with its default values.
+# The data can then be loaded with the rails db:seed command (or created
+# alongside the database with db:setup).
 #
 # Examples:
 #
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   movies = Movie.create([{ name: 'Star Wars' },
+#                          { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+bob = User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+sarah = User.create(username: 'Sarah', email: 'sarah@mail.com', password: '321')
+ivan = User.create(username: 'Ivan', email: 'breku@mail.com', password: 'ur')
```

Comment:
Just thinking if you ever want to sign in as these users it may be nice to have a high level of consistency with their emails and password. I notice all other Users emails follow a pattern besides this one. Also you could probably make all the passwords the same.